### PR TITLE
fix max char limit

### PR DESCRIPTION
**Pull request by SirJosh3917**
it's actually 1014 sorry guys

### CODE REVIEW

File: Modix.Bot/Modules/ReplModule.cs [hunk 1]
```diff
@@ -35,8 +35,11 @@ public class ReplModule : ModuleBase
         // 1024 is the maximum field embed size
         // minus 3 for the start codeblocks
         // minus 3 for the end codeblocks
+        // minus 1 for the first newline after the ```cs
+        // minus 2 for 'cs'
+        // minus 1 for the next newline after the code
         // THIS LIMIT IS NOT ARBITRARY
-        private const int MaxReplSize = 1_024 - 3 - 3;
+        private const int MaxReplSize = 1_024 - 3 - 3 - 1 - 2 - 1;
```
Comment (Scott-Caldwell) [inline Modix.Bot/Modules/ReplModule.cs]: This is getting a little hard to parse. Would it be better to just set it to 1014 with a comment?

Comment (SirJosh3917) [reply Modix.Bot/Modules/ReplModule.cs]: hard to parse? i think each comment in order of why we're removing chars and then removing it in the code is fine

Comment (calledude) [reply Modix.Bot/Modules/ReplModule.cs]: Each comment explains clearly why it should be set to 1014, doing it this way is extremely verbose, which is what I think @Scott-Caldwell was getting at.

Comment (jmazouri) [reply Modix.Bot/Modules/ReplModule.cs]: I'm ok with composing it like this. It is a bit awkward to look at but the comment provides the necessary clarity.
